src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationRepository.cs

```diff
@@ -169,4 +169,16 @@ public async Task<IEnumerable<string>> GetOwnerEmailAddressesById(Guid organizat
             new { OrganizationId = organizationId },
             commandType: CommandType.StoredProcedure);
     }
+
+    public async Task EnableCollectionEnhancements(Guid id)
+    {
+        using (var connection = new SqlConnection(ConnectionString))
+        {
+            await connection.ExecuteAsync(
+                "[dbo].[Organization_EnableCollectionEnhancements]",
+                new { Id = id },
```

Comment:
I have also named the parameter `OrganizationId` on the sproc
```suggestion
                new { OrganizationId = organizationId },
```